src/EnergyPlus/PsychCacheData.hh

```diff
@@ -136,6 +136,12 @@ struct PsychrometricCacheData : BaseGlobalStruct
     Array1D<cached_tsat_h_pb> cached_Tsat_HPb; // DIMENSION(0:tsat_hbp_cache_size)
 #endif
 
+#ifdef EP_psych_stats
+    // EnergyPlus::Psychrometrics::NumPsychMonitors = 19
+    Array1D<Int64> NumTimesCalled = Array1D<Int64>(19, 0);
```

Comment:
@Myoldmopar this could use a review. I tried including ```Psychrometrics.hh``` in this file so I could create the array using the reference to ```NumPsychMonitors```, but there must still be something else that is needed. I just hardcoded the number for now to get it going, but I'm happy to change it if you can point me in the right direction.